model-optimizer/mo/main.py

```diff
@@ -254,9 +256,22 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
 
     if not (argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update):
         output_dir = argv.output_dir if argv.output_dir != '.' else os.getcwd()
-        print('\n[ SUCCESS ] Generated IR version {} model.'.format(get_ir_version(argv)))
-        print('[ SUCCESS ] XML file: {}.xml'.format(os.path.join(output_dir, argv.model_name)))
-        print('[ SUCCESS ] BIN file: {}.bin'.format(os.path.join(output_dir, argv.model_name)))
+        orig_model_name = os.path.normpath(os.path.join(output_dir, argv.model_name))
+
+        if not find_ie_version():
+            print("[ WARNING ] No InferenceEngine python was found. Some ModelOptimizer functionality may not work.")
+            print("[ WARNING ] Please consider to build InferenceEngine python from source or try to install OpenVINO using install_prerequisites.{}".format(
+                "bat" if sys.platform == "windows" else "sh"))
+        else:
+            path_to_offline_transformations = os.path.join(os.path.realpath(os.path.dirname(__file__)),
+                                                           'offline_transformations.py')
+            status = subprocess.run([sys.executable, path_to_offline_transformations, orig_model_name], env=os.environ, timeout=100)
```

Comment:
Does 100 seconds is a proper timeout? For me it looks too much.

Reply:
Right, for testing purposes I think we ca decrease it. Do you have any suggestions? 5s?

Reply:
Changed to 5s.